analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -5,46 +5,53 @@
 
     <!-- Copy project-specific SonarLint.xml to enable project-specific settings, such as file headers -->
     <PropertyGroup>
-      <ProjectSpecificSonarLintXml Condition="Exists('$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml')">
-        $(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml
-      </ProjectSpecificSonarLintXml>
-      <ProjectSpecificSonarLintXml Condition="!Exists('$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml')">
-        $(MSBuildStartupDirectory)\config\SonarLint.xml
-      </ProjectSpecificSonarLintXml>
+      <ProjectSpecificSonarLintXml Condition="Exists('$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml')">$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml</ProjectSpecificSonarLintXml>
+      <ProjectSpecificSonarLintXml Condition="!Exists('$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml')">$(MSBuildStartupDirectory)\config\SonarLint.xml</ProjectSpecificSonarLintXml>
     </PropertyGroup>
 
     <PropertyGroup>
       <SonarAnalyzerRegressionTestRunning Condition="Exists('$(MSBuildStartupDirectory)\binaries\SonarAnalyzer.dll')">true</SonarAnalyzerRegressionTestRunning>
       <SonarAnalyzerRegressionTestRunning Condition="$(SonarAnalyzerRegressionTestRunning) == ''">false</SonarAnalyzerRegressionTestRunning>
-      <ProjectConfFilePath>
-        $(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)\ProjectOutFolderPath.txt;$(ProjectSpecificSonarLintXml)
-      </ProjectConfFilePath>
+      <SonarProjectOutputPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)</SonarProjectOutputPath>
+      <SonarProjectConfigFilePath>$(SonarProjectOutputPath)\SonarProjectConfig.xml</SonarProjectConfigFilePath>
     </PropertyGroup>
 
+    <!-- This section builds content of SonarProjectConfig.xml additional file -->
+    <ItemGroup>
+      <SonarProjectConfigLine Include="&lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;" />
```

Comment:
ProjectType seems to be missing? if it's intentional, please add a comment to mention it

Reply:
It was intentional to let analyzer take the decision. But that's not how it will work after MMF-2298 anyway so it's ready for the future now.